mcs/class/System/Test/System.Net.Security/SslStreamTest.cs

```diff
@@ -93,6 +171,167 @@ void AuthenticateClientAndServer (bool server, bool client)
 		}
 	}
 
+	[Category("NotWorking")]    // Works in .Net, doesn't work in mono.
+	[Test()]
+	public void SelfCompatibilityTest()
+	{
+		#if __MonoCS__
```

Comment:
@marek-safar That won't help here as e.g. the `MonoX509Certificate` class doesn't exist on MS.NET (the intention by @rahvee was that the test could be compiled&run on MS.NET as well).